test/browser/features/DVA_Driving_Licence.feature

```diff
@@ -15,17 +15,19 @@ Feature: DVA Driving licence CRI Error Validations
     When User clicks on continue
     Then I see the DVA licence number error in the summary as Your licence number should be 8 characters long
     And I can see the DVA licence number error in the field as Your licence number should be 8 characters long
+    And I check the page Title Error: Enter your details exactly as they appear on your UK driving licence – Prove your identity – GOV.UK
     Examples:
       |DVADrivingLicenceSubject       |InvalidLicenceNumber|
       |DrivingLicenceSubjectHappyBilly|5566778             |
 
-  @mock-api:dva-invalidDrivingLicenceNumber @validation-regression @build @staging
+  @mock-api:dva-invalidDrivingLicenceNumberWithSplChar
```

Comment:
Why are we removing tags here?

Reply:
Changed to the existing tag and committed the changes.